spec/sneakers/worker_handlers_spec.rb

```diff
@@ -103,6 +103,31 @@ def process(*args,&block)
 
   describe 'Maxretry' do
     let(:max_retries) { nil }
+    let(:props_with_x_death_count) {
+      {
+        :headers => {
+          "x-death" => [
+                        {
+                          "count" => 3,
+                          "reason" => "expired",
+                          "queue" => "downloads-retry",
+                          "time" => Time.now,
+                          "exchange" => "RawMail-retry",
+                          "routing-keys" => ["RawMail"]
+                        },
+                        {
+                          "count" => 3,
```

Comment:
Yeah, that makes sense. I think it's better to keep the test data more realistic, so what you have here is great. Thanks for the explanation. It's been awhile since I've looked into this stuff.